### PR TITLE
Redesign popular tests section

### PR DESCRIPTION
**Pull request by karinabarinova**
https://trello.com/c/9CGxWgiL/80-redesign-popular-tests-section
Screenshot 2023-06-18 at 21 54 58
Screenshot 2023-06-18 at 22 09 06
Screenshot 2023-06-18 at 22 09 23

### CODE REVIEW

**Review by Flowerinno — APPROVED** [review]
bomba

File: src/components/elements/Text/Text.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ interface TextProps {
     isParagraph?: boolean;
     size?: "big" | "medium" | "small";
     color?: "darkTextColor" | "lightTextColor" | "greyTextColor" | "mostLightColor";
-    bold?: true | false;
+    semiBold?: true | false;
```
Comment (kas1qqqq) [inline src/components/elements/Text/Text.tsx]:
``` suggestion
    semiBold?: boolean;
```

File: src/components/elements/Button/Button.module.scss [hunk 2]
```diff
@@ -7,9 +7,8 @@
     display: inline-flex;
     align-items: center;
     justify-content: center;
-    font-family: $primary-font;
+    font-family: inherit;
```
Comment (kas1qqqq) [inline src/components/elements/Button/Button.module.scss]: will come globally in a future PR
```suggestion

File: src/components/elements/Button/Button.module.scss [hunk 3]
```diff
@@ -7,9 +7,8 @@
...
     font-style: normal;
```
Comment (kas1qqqq) [inline src/components/elements/Button/Button.module.scss]:
```suggestion

File: src/components/elements/Error/Error.module.scss [hunk 4]
```diff
@@ -7,20 +7,18 @@
     align-items: center;
     margin: 0.5rem 2rem;
     padding: 1.9rem 1.2rem;
-    background-color: $bg-error-color;
+    background-color: $error-bg;
     border-radius: 1.6rem;
-    font-family: $primary-font;
+    font-family: $fracktif-semi-bold;
     font-style: normal;
```
Comment (kas1qqqq) [inline src/components/elements/Error/Error.module.scss]: I’m not sure about necessary it
```suggestion

File: src/components/modules/Footer/NavigationList.module.scss [hunk 5]
```diff
@@ -16,9 +15,8 @@
                 flex-wrap: nowrap;
                 text-align: center;
                 font-style: normal;
```
Comment (kas1qqqq) [inline src/components/modules/Footer/NavigationList.module.scss]:
```suggestion

File: src/components/modules/Header/Auth.module.scss [hunk 6]
```diff
@@ -23,9 +23,7 @@
 
 .navigationItem {
     padding: 0 1.6rem;
-    font-family: $primary-font;
     font-style: normal;
```
Comment (kas1qqqq) [inline src/components/modules/Header/Auth.module.scss]:
```suggestion

File: src/components/modules/Header/Header.module.scss [hunk 7]
```diff
@@ -9,9 +9,8 @@
     justify-content: center;
     width: 100%;
     height: 8rem;
-    background: $most-light-color;
+    background: $white;
     box-shadow: 0 1px 4px rgb(0 0 0 / 25%);
-    font-family: $primary-font;
     z-index: 999;
```
Comment (kas1qqqq) [inline src/components/modules/Header/Header.module.scss]: if enough
```suggestion
    z-index: 1 or 2;
```

File: src/components/modules/Header/Navigation.module.scss [hunk 8]
```diff
@@ -8,9 +8,7 @@
 
 .navigationItem {
     padding: 0 2.4rem;
-    font-family: $primary-font;
     font-style: normal;
```
Comment (kas1qqqq) [inline src/components/modules/Header/Navigation.module.scss]:
```suggestion

File: src/components/modules/TestsSection/TestsSectionDesktop.module.scss [hunk 9]
```diff
@@ -207,9 +239,9 @@
     height: max-content;
     font-size: 1rem;
     line-height: 2.5rem;
-    color: #16161F;
+    color: $primary-text-dark;
     font-weight: 600;
```
Comment (kas1qqqq) [inline src/components/modules/TestsSection/TestsSectionDesktop.module.scss:243]:
```suggestion

File: src/components/modules/TestsSection/TestsSectionMobile.module.scss [hunk 10]
```diff
@@ -105,15 +119,16 @@
     margin-bottom: 2.4rem;
     padding: 0.8rem 0;
     font-weight: 500;
```
Comment (kas1qqqq) [inline src/components/modules/TestsSection/TestsSectionMobile.module.scss:121]:
```suggestion

File: src/pages/all-tests.tsx [hunk 11]
```diff
@@ -24,17 +24,6 @@ const AllTestsPage: NextPage<PageProps> = ({ allTests, error }: PageProps) => {
...
-
-    if (!accessToken) {
-        return {
-            redirect: {
-                destination: "/auth/sign-in",
-                permanent: false,
-            },
-        };
-    }
-
```
Comment (kas1qqqq) [inline src/pages/all-tests.tsx:37]:
```suggestion
why did you delete it?

File: src/components/elements/Logo/Logo.module.scss [hunk 12]
```diff
@@ -3,22 +3,21 @@
 
 .logo {
     & > button {
-        font-family: $fracktif-medium;
+        font-family: $fracktif-semi-bold;
         font-style: normal;
```
Comment (kas1qqqq) [inline src/components/elements/Logo/Logo.module.scss]:
```suggestion

**Review by kas1qqqq — CHANGES_REQUESTED** [review]

File: src/pages/all-tests.tsx (hunk 11 above)
Comment (karinabarinova) [reply src/pages/all-tests.tsx:37]: We're able to get the tests data without authentication now.

File: src/components/elements/Text/Text.tsx (hunk 1 above)
Comment (karinabarinova) [reply src/components/elements/Text/Text.tsx]: These changes can be applied with a separate MR, as it does not concern the changes of this task.

File: src/components/elements/Button/Button.module.scss (hunk 2 above)
Comment (karinabarinova) [reply src/components/elements/Button/Button.module.scss]: [Same](https://github.com/Interviewboom/frontend/pull/41#discussion_r1240913302)

File: src/components/elements/Button/Button.module.scss (hunk 3 above)
Comment (karinabarinova) [reply src/components/elements/Button/Button.module.scss]: [Same](https://github.com/Interviewboom/frontend/pull/41#discussion_r1240913302)

File: src/components/elements/Error/Error.module.scss (hunk 4 above)
Comment (karinabarinova) [reply src/components/elements/Error/Error.module.scss]: [Same](https://github.com/Interviewboom/frontend/pull/41#discussion_r1240913302)

File: src/components/modules/Footer/NavigationList.module.scss (hunk 5 above)
Comment (karinabarinova) [reply src/components/modules/Footer/NavigationList.module.scss]: [Same](https://github.com/Interviewboom/frontend/pull/41#discussion_r1240913302)

File: src/components/modules/Header/Auth.module.scss (hunk 6 above)
Comment (karinabarinova) [reply src/components/modules/Header/Auth.module.scss]: [Same](https://github.com/Interviewboom/frontend/pull/41#discussion_r1240913302)

File: src/components/modules/Header/Header.module.scss (hunk 7 above)
Comment (karinabarinova) [reply src/components/modules/Header/Header.module.scss]: [Same](https://github.com/Interviewboom/frontend/pull/41#discussion_r1240913302)

File: src/components/modules/Header/Navigation.module.scss (hunk 8 above)
Comment (karinabarinova) [reply src/components/modules/Header/Navigation.module.scss]: [Same](https://github.com/Interviewboom/frontend/pull/41#discussion_r1240913302)

File: src/components/modules/TestsSection/TestsSectionDesktop.module.scss (hunk 9 above)
Comment (karinabarinova) [reply src/components/modules/TestsSection/TestsSectionDesktop.module.scss:243]: [Same](https://github.com/Interviewboom/frontend/pull/41#discussion_r1240913302)

File: src/components/modules/TestsSection/TestsSectionMobile.module.scss (hunk 10 above)
Comment (karinabarinova) [reply src/components/modules/TestsSection/TestsSectionMobile.module.scss:121]: [Same](https://github.com/Interviewboom/frontend/pull/41#discussion_r1240913302)

File: src/components/elements/Logo/Logo.module.scss (hunk 12 above)
Comment (karinabarinova) [reply src/components/elements/Logo/Logo.module.scss]: [Same](https://github.com/Interviewboom/frontend/pull/41#discussion_r1240913302)

**Review by kas1qqqq — APPROVED** [review]

**Pull request merged**